python/lsst/ci/hsc/validate.py

```diff
@@ -189,6 +189,14 @@ class SfmValidation(Validation):
     def validateSources(self, dataId):
```

Comment:
I would recommend a more descriptive name for this test since it is testing extendedness.